libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/stepper-advanced.service.ts

```diff
@@ -0,0 +1,392 @@
+import { Injectable } from '@angular/core';
+import { SdsFormlyTypes } from '@gsa-sam/sam-formly';
+import { FormlyFieldConfig } from '@ngx-formly/core';
+
+@Injectable()
+export class StepperAdvancedService {
+  getPermission(): FormlyFieldConfig {
+    return {
+      key: 'permissions',
+      fieldGroupClassName: 'grid-row',
+      fieldGroup: [
+        {
+          key: 'permissionsNeeded',
+          template: '<br/><h4><b>what permission</b></h4></br> Select the specfific <br>',
+        },
+        {
+          key: 'typeOfEntity',
+          className: 'desktop: grid-col-12 tablet: grid-col-12',
+          type: SdsFormlyTypes.RADIO,
+          templateOptions: {
+            label: 'What type of entity are you registering?',
+            hideOptions: true,
+            options: [
+              {
+                label: 'Business Or Organization',
+                value: 'business',
+              },
+              {
+                label: 'U.S. State Government',
+                value: 'stateGovt',
+              },
+              {
+                label: 'U.S. Local Government',
+                value: 'localGovt',
+              },
+              {
+                label: 'Tribal Government',
+                value: 'tribal',
+              },
+              {
+                label: 'Foreign Government',
+                value: 'foreign',
+              },
+            ],
+          },
+        },
+        {
+          key: 'contractOpp',
+          type: SdsFormlyTypes.MULTICHECKBOX,
+          className: 'desktop: grid-col-12 tablet: grid-col-12',
+          templateOptions: {
+            label: 'Contract Opportunities',
+            group: 'panel',
+            options: [
+              {
+                key: 'read-public',
+                value: 'Read Public',
+              },
+              {
+                key: 'read-sensitive',
+                value: 'Read sensitive',
+              },
+              {
+                key: 'write',
+                value: 'Write public',
+              },
+              {
+                key: 'write-sensitive',
+                value: 'Write sensitive',
+              },
+            ],
+          },
+        },
+      ],
+    };
+  }
+
+  getRegistrationPurpose(): FormlyFieldConfig {
+    return {
+      key: 'purposeOfRegistration',
+      fieldGroup: [
+        {
+          key: 'typeOfEntity',
+          type: SdsFormlyTypes.RADIO,
+          templateOptions: {
+            label: 'What type of entity are you registering?',
+            hideOptions: true,
+            options: [
+              {
+                label: 'Business Or Organization',
```

Comment:
Shouldn't this be key: value with the latest formly version?

Reply:
new update uses label, key was used previously.